roles/ocitysmap/vars/main.yml

```diff
@@ -0,0 +1,72 @@
+locale_list:
+  - 'fr_FR ISO-8859-1'
+  - 'fr_FR.UTF-8 UTF-8'
+  - 'fr_FR@euro ISO-8859-15'
+  - 'de_DE ISO-8859-1'
+  - 'de_DE.UTF-8 UTF-8'
+  - 'de_DE@euro ISO-8859-15'
+  - 'en_GB ISO-8859-1'
+  - 'en_GB.ISO-8859-15 ISO-8859-15'
+  - 'en_GB.UTF-8 UTF-8'
+  - 'en_US ISO-8859-1'
+  - 'en_US.ISO-8859-15 ISO-8859-15'
+  - 'en_US.UTF-8 UTF-8'
+debian_python_required_packages:
+  #Ce module ne gere pas les extensions Cairo, on va l'installer manuellement plus loin
+  - python-mapnik
+  - python-psycopg2
+  - python3-psycopg2
```

Comment:
Are we using both python 2 and 3 ?

Reply:
i try to use python3 when it's possible
i'll check that point ...